src/backend/alembic/versions/a48691a80366_.py

```diff
@@ -0,0 +1,74 @@
+"""empty message
+
+Revision ID: a48691a80366
+Revises: 2d766d30a3d2
+Create Date: 2024-06-28 17:09:51.378370
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "a48691a80366"
+down_revision: Union[str, None] = "2d766d30a3d2"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    # op.alter_column('agent_tool_metadata', 'artifacts',
+    #            existing_type=postgresql.ARRAY(sa.TEXT()),
+    #            type_=postgresql.ARRAY(postgresql.JSONB(astext_type=sa.Text())),
+    #            existing_nullable=True)
+
+    op.drop_column("agent_tool_metadata", "artifacts")
+    op.add_column(
+        "agent_tool_metadata",
+        sa.Column(
+            "artifacts",
+            postgresql.ARRAY(postgresql.JSONB(astext_type=sa.Text())),
+            nullable=True,
+        ),
+    )
```

Comment:
I had to manually alter this. Changing the column type in the alembic db_model file will fail since there it exists as TEXT[] and you can't marshal it to JSON[] without doing something in the script itself here. 

Should be fine when we deploy since this feature isn't used yet. Lmk otherwise.

@lusmoura @tianjing-li